server/template/paste.xhtml

```diff
@@ -0,0 +1,61 @@
+<html py:extends="master.xhtml" py:strip="True">
+<body>
+
+  <head py:block="head" py:strip="True">
+    <title py:block="master_title">Paste Logs</title>
+  <style type="text/css">
+   .container {
+       width: 1000px;
+       clear: both;
+   }
+   .container input {
+       width: 100%;
+       clear: both;
+   }
+  </style>
+</head>
+
+<body py:block="body" py:strip="True">
+  <div class="container">
+    <form name="upload_form" action="paste" method="post" enctype="multipart/form-data">
+    <div class="form-group" id="message">
+      <textarea name="message" rows="10" cols="60" placeholder="Enter logs here or upload a text file below" >${raw_logs}</textarea>
+      <br /><br />
+      <input type="file" name="upload" accept="*.txt" ></input>
+      <div class="radio">
+        <label>
+          <input type="radio" name="os_type" value="iOS" checked="checked" /> iOS
+        </label>
+      </div>
+      <div class="radio">
+        <label>
+          <input type="radio" name="os_type" value="Android" /> Android
+        </label>
+      </div>
+      <input value="Submit" type="submit" style="height:30px; width:80px" />
+    </div>
+  </form>
+  </div>
+  <div class="row">
+    <div class="col-md-8">
+      <p></p>
+      <table class="table table-hover table-condensed table-bordered log-table" id="log-table">
+        <thead>
+          <tr>
+            <th>Time</th>
+            <th>Tag</th>
+            <th>Type</th>
+            <th>Log Text</th>
+          </tr>
+        </thead>
+        <tbody id="log-table-body-paste">
+          <!-- use kajiki to include rendered html -->
```

Comment:
remove or update comment to something like: "Use Kajiki to render the table rows"